app/out-of-office/server/lib/outOfOfficeUtils.ts

```diff
@@ -0,0 +1,26 @@
+import { OutOfOfficeRooms, OutOfOfficeUsers } from '../../../models/server';
+
+export async function addToOutOfOfficeRoomsCollection(
+	userId: string,
+	roomIds: string[],
+): Promise<void> {
+	await Promise.all(
+		roomIds.map(async (roomId) =>
+			OutOfOfficeRooms.addUserIdAndRoomId({ userId, roomId }),
+		),
+	).catch((e) => {
+		console.log(
```

Comment:
Please use `Logger` instead of `console.log`

Reply:
Refactored to `Logger`!